routes/events.js

```diff
@@ -47,4 +56,18 @@ module.exports = function(app) {
             }
         });
     });
+
+    // Route for retrieving data of all events in a tier
+    app.get('/events/:tier', function(req, res, next) {
+
+        eventServices.findAllEventsInTier(function(err, events) {
+            if (err) {
+                console.log(err);
+                res.sendStatus(500);
+            } else {
+                // TODO: Send  what?
+                res.send(post/postt_tier', {post: posts'};
```

Comment:
if you're passing an object to `res.send`, it actually does the same thing as `res.json`. nbd, just FYI.